packages/node_modules/@node-red/editor-client/src/js/ui/clipboard.js

```diff
@@ -940,28 +940,25 @@ RED.clipboard = (function() {
         if (truncated) {
             msg += "_truncated";
         }
-        $("#red-ui-clipboard-hidden").val(value).focus().select();
-        var result =  document.execCommand("copy");
-        if (result && element) {
-            var popover = RED.popover.create({
-                target: element,
-                direction: 'left',
-                size: 'small',
-                content: RED._(msg)
-            });
-            setTimeout(function() {
-                popover.close();
-            },1000);
-            popover.open();
-        }
-        $("#red-ui-clipboard-hidden").val("");
-        if (currentFocus) {
-            $(currentFocus).focus();
-        }
-        return result;
+        navigator.clipboard.writeText(value).then(function () {
+            if (element) {
+                var popover = RED.popover.create({
+                    target: element,
+                    direction: 'left',
+                    size: 'small',
+                    content: RED._(msg)
+                });
+                setTimeout(function() {
+                    popover.close();
+                },1000);
+                popover.open();
+            }
+            if (currentFocus) {
+                $(currentFocus).focus();
+            }
+        });
```

Comment:
Need to handle a rejected promise here. For now, suggest the following:
```suggestion
        }).catch(err => { console.error("Failed to copy:",err) });
```

Reply:
Thank you for your adding the code to handle the error. I accepted it.